app.js

```diff
@@ -36,168 +36,178 @@ function promptUserName(){
   }while (!correctName);
 }
 
-var wantsToPlay = confirm('Nice to meet you ' + userName + '! Would you like to take a quick quiz about me? (click OK to continue.)');//confirm user wants to play game
+function wantsToPlay(){
+  var wantToPlay = confirm('Nice to meet you ' + userName + '! Would you like to take a quick quiz about me? (click OK to continue.)');//confirm user wants to play game
```

Comment:
Good catch on not reusing the name of the function as this variable name. Personally, I like my functions that do something (like `prompt`/`confirm`) to have verb names, so I probably would have left this variable as-is and named the function `confirmWantsToPlay`.